src/todolist.test.js

```diff
@@ -0,0 +1,60 @@
+describe('tests',() => {
+    test('addtodo test',() =>{
+        const toadd = 'laundry';
+        const tasks = [
+            {description:'task1',completed:false,index:0},
+            {description:'task2',completed:false,index:1},
+            {description:'task3',completed:false,index:2}
+        ]
+       function addTask(task, taskarray){
+            taskarray.push({
+              description: task,
+              completed: false,
+              index: 1
+            });
+            return taskarray.length;
+          }
+    expect(addTask(toadd,tasks)).toBe(4)
+    })
```

Comment:
- [ ] good job wrapping the tests inside the description block 👍 however you are not actually testing your code you are in this case just testing the functions you wrote inside your tests.
- [ ] additionally you are creating the `task` constant in every test, try applying the DRY rules by creating it once inside your description block and if neseccary resetting it before every test by using the `beforeEach()`